pysiaf/utils/polynomial.py

```diff
@@ -545,7 +545,7 @@ def prepend_rotation_to_polynomial(a, theta, verbose=False):
                 for j in range(m-n-mu, m-mu+1):
                     factor = (-1)**(m-n-mu) * choose(m-j, mu) * choose(j, m-n-mu)
                     cosSin = c**(j+2*mu-m+n) * s**(2*m-2*mu-j-n)
-                    atrotate[m, n] = atrotate[m, n] + factor * cosSin * at[m, j]
+                    atrotate[m, n] = np.squeeze(atrotate[m, n] + factor * cosSin * at[m, j])
```

Comment:
I am looking at a two-dimensional array in python:

```
>>>: x = np.random.rand(3,3)
>>> print(x)

array([[0.8991704 , 0.47865258, 0.24120141],
       [0.14969249, 0.59154556, 0.9030803 ],
       [0.46856477, 0.45624738, 0.92141728]])

>>> x.squeeze()

array([[0.8991704 , 0.47865258, 0.24120141],
       [0.14969249, 0.59154556, 0.9030803 ],
       [0.46856477, 0.45624738, 0.92141728]])
```

It looks like the squeeze method doesn't change the shape/dimensions of the array, is this change necessary?